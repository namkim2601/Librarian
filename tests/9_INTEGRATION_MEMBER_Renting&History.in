MeMbEr BoOkS nomembers
MeMbEr HiStOry nomembers
add member Nam Kim
member books nosuchmember
member history nosuchmember
member books 100000
member history 100000
add collection collection.csv
rent 100000 111113
rent 100000 111112
rent 100000 111111
member books 100000
relinquish 100000 111112
member books 100000
relinquish all 100000
member books 100000
member history 100000
exit
