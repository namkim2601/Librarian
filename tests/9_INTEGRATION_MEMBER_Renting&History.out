user: No members in system.

user: No members in system.

user: Success.

user: No such member in system.

user: No such member in system.

user: Member not currently renting.

user: No rental history for member.

user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: The Iliad (Homer)
The Hobbit (J.R.R. Tolkien)
To Kill a Mockingbird (Harper Lee)

user: Success.

user: The Iliad (Homer)
To Kill a Mockingbird (Harper Lee)

user: Success.

user: Member not currently renting.

user: The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
To Kill a Mockingbird (Harper Lee)

user: Ending Library process.
